pkgs/desktops/gnome-3/extensions/system-monitor/default.nix

```diff
@@ -1,34 +1,41 @@
-{ stdenv, fetchFromGitHub, glib }:
+# This package needs the following configuration in configuration.nix:
+# services.xserver.desktopManager.gnome3.sessionPath = [ pkgs.libgtop pkgs.glib_networking ];
+{ config, stdenv, fetchFromGitHub, glib, glib_networking, libgtop, pkgs }:
 
 stdenv.mkDerivation rec {
   name = "gnome-shell-system-monitor-${version}";
-  version = "8b31f070e9e59109d729661ced313d6a63e31787";
+  version = "v33";
```

Comment:
For Nix to understand the version, it must start with a number. (Change this one to "33" and set ```rev = "v${version}";``` below.)